main.go

```diff
@@ -99,15 +96,29 @@ Flags:
 	flags.StringVar(&opts.kubeConfigPath, "kc", os.Getenv("HOME")+"/.kube/", "set cluster kubeconfig path")
 	flags.StringVar(&opts.outputFile, "o", "/cluster-logs/", "set logs output file")
 	flags.BoolVar(&opts.version, "version", false, "show version and exit")
-	flags.BoolVar(&opts.diff, "diff", false, "create diff file")
+	flags.BoolVar(&opts.pod, "pod", false, "extract pod logs option")
+	flags.BoolVar(&opts.cm, "cm", false, "extract config maps option")
+	flags.BoolVar(&opts.svc, "svc", false, "extract services option")
+	flags.BoolVar(&opts.crd, "crd", false, "extract crds option")
+
+	if !opts.pod && !opts.cm && !opts.svc && !opts.crd {
```

Comment:
Can you please explain about why this if statement is needed?

Reply:
If no specific option is given then all options will be activated

Reply:
Wouldn't is be better to have the default value be true instead of false if that is your desired behavior?